libcodechecker/analyze/log_parser.py

```diff
@@ -106,7 +106,13 @@ def dump_compiler_info(output_path, filename, data):
         f.write(json.dumps(all_data))
 
 
-def get_compiler_includes(compiler, lang, compile_opts, output_path=None,
+def load_compiler_info(filename, compiler):
+    with open(filename, 'r') as f:
+        data = json.load(f)
+    return data[compiler]
```

Comment:
This will throw a KeyError exception if the key is missing. I prefer to use `.get(key)` which will return `None` by default if the key is missing or a default value can be returned too if needed.

Reply:
Ok, I changed it to `.get`. If it is `None` we log an error.